build.sh

```diff
@@ -157,7 +157,7 @@ if hasArg clean; then
         if [ -d "${bd}" ]; then
             find "${bd}" -mindepth 1 -delete
             rmdir "${bd}" || true
-	fi
+    fi
```

Comment:
```suggestion
    fi
```
Seems like a misaligned `fi`?

Reply:
Ack thanks. In removing tabs I accidentally deindented this.